sources/packages/web/src/assets/css/base.scss

```diff
@@ -377,6 +377,13 @@ ul {
   }
 }
 
+// List style - apply to ordered lists used in content
```

Comment:
This will change all the places with `ol > li. Is that the purpose? Wouldn't it break other layouts?

Reply:
Thank You @andrepestana-aot. This was the cleanup code that I missed to remove before raising the PR.